EventHubs/tests/baseeventhubtest.py

```diff
@@ -1,28 +1,27 @@
 import os
 import sys
-sys.path.insert(0, '../../test_utils')
-from basetest import BaseTest
 import json
+import time
 from time import sleep
 from requests import Session
+from datetime import datetime
+from azure.servicebus import ServiceBusService
 from azure.mgmt.resource import ResourceManagementClient
 from azure.mgmt.eventhub import EventHubManagementClient
-from azure.mgmt.storage import StorageManagementClient
-from azure.cosmosdb.table.tableservice import TableService
-from azure.servicebus import ServiceBusService
 
+sys.path.insert(0, '../../test_utils')
+from basetest import BaseTest
 
 class BaseEventHubTest(BaseTest):
 
     def setUp(self):
         self.create_credentials()
-        self.resource_client = ResourceManagementClient(self.credentials,
-                                                        self.subscription_id)
+        self.resource_client = ResourceManagementClient(self.azure_credential, self.subscription_id)
         try:
-            self.sumo_endpoint_url = os.environ["SumoEndpointURL"]
+            self.sumo_endpoint_url = os.getenv("SumoEndpointURL")
```

Comment:
revert back if you are using get then it won't throw KeyError

Reply:
updated